.github/workflows/cleanup-review-apps.yml

```diff
@@ -0,0 +1,24 @@
+name: Cleanup Review Apps
+on:
+  schedule:
+    - cron: '0 7 * * *'
+    # At 07:00, daily
```

Comment:
Apps we have created will exist until we delete them and running them will cost some non-zero amount of money. We need some way to delete inactive apps.

The way I've done this, I've just made a scheduled action that runs once a day and deletes any review apps that exist. This means in some cases, we will delete review apps for in-progress PRs so we'll have to re-create the review app. I think this is OK. It only takes a few minutes to run the workflow create a review app, even from scratch. There's a number of ways we could tweak this: We could run it less frequently. We could only delete review apps for closed PRs. I reckon probably run this for a month and see what the bill looks like then review if we want to tweak it.